js/screen-saver.js

```diff
@@ -0,0 +1,46 @@
+var app = app || {};
+
+app.screenSaver = (function() {
+
+	'use strict';
+
+	return {
+
+		screenSaverActive: false,
+
+		initialize: function() {
+
+			this.createTimer();
+
+			$(document).click(_.bind(function(){
```

Comment:
The event callback function here should be a method defined on the class/prototype. In this case, a name like `resetTimer` could be good.